### PR TITLE
ER-1414 entity changes for revoke

### PR DESCRIPTION
**Pull request by dev4valtech**

### CODE REVIEW

File: src/Shared/Recruit.Vacancies.Client/Domain/Entities/BlockedOrganisation.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public class BlockedOrganisation
+    {
+        public Guid Id { get; set; }
+        public OrganisationType OrganisationType { get; set; }
+        public string OrganisationId { get; set; }
+        public BlockedStatus BlockedStatus { get; set; }
+        public VacancyUser UpdatedBy { get; set; }
```
Comment (sachpatel) [inline src/Shared/Recruit.Vacancies.Client/Domain/Entities/BlockedOrganisation.cs]: `UpdatedByUser`

File: src/Shared/Recruit.Vacancies.Client/Domain/Entities/TransferInfo.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public class TransferInfo
+    {
+        public long Ukprn { get; set; }
+        public string ProviderName { get; set; }
```
Comment (sachpatel) [inline src/Shared/Recruit.Vacancies.Client/Domain/Entities/TransferInfo.cs]: I believe we should also include the `LegalEntityId` in this class unless it is the same as the one in the top level of the Vacancy document in which case `LegalEntityName` would be redundant.

Comment (dev4valtech) [reply src/Shared/Recruit.Vacancies.Client/Domain/Entities/TransferInfo.cs]: Legal entity can be derived from Vacancy, so I am not sure why the name was included, perhaps to make it easy to project/query, perhaps @mgwilliam can tell?

Comment (mgwilliam) [reply src/Shared/Recruit.Vacancies.Client/Domain/Entities/TransferInfo.cs]: @dev4valtech yes it's partly for that and also if the employer subsequently changes the legal entity - so it's basically a snapshot of the legal entity at the time of the transfer

Comment (dev4valtech) [reply src/Shared/Recruit.Vacancies.Client/Domain/Entities/TransferInfo.cs]: @mgwilliam does it make sense to have legal entity id as well then?

**Pull request merged**